<launch>
  <node pkg="irb140_commander" type="joint_commander" name="joint_commander" args="irb140_arm" output="screen" />
  <node pkg="irb140_commander" type="pose_commander" name="pose_commander" args="irb140_arm" output="screen" />
  <!-- <node pkg="irb140_commander" type="path_commander" name="path_commander" output="screen"/> -->
</launch>